diff --git a/dist/index.js b/dist/index.js
index 374c299db7f94eff8a8a6db859e7360171439d7f..16f362931ae675858b798b2046afcb0b7ba86b99 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -79,7 +79,7 @@ var getParent = ({ opts }) => {
     }
   });
   const parentPage = parentPages.reverse().find(
-    (page) => "frontMatter" in page && page.frontMatter && page.frontMatter.type === "posts"
+    (page) => "frontMatter" in page && page.frontMatter && ( page.frontMatter.type === "posts" || page.frontMatter.type === "page" )
   );
   if (parentPage) {
     back = parentPage.route;